modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -290,10 +293,10 @@ public Response getBatchAssignments(@PathParam("applicationName")
                     headers, experimentBatch);
         }
 
-        List<Map> myAssignments = assignments.doBatchAssignments(userID, applicationName, context, createAssignment, FALSE,
-                headers, experimentBatch, null, null);
+        List<Assignment> myAssignments = assignments.doBatchAssignments(userID, applicationName, context, createAssignment, FALSE,
```

Comment:
Cosmetic: Suggest changing variable name to `batchAssignments` instead of `myAssignments`.